.github/workflows/ci.yaml

```diff
@@ -33,18 +31,10 @@ jobs:
       - name: Install dependencies
```

Comment:
use shorter semantic names for steps. `install` is enough. always prefer lowercase
`Linting -> lint`. Use verbs. like you step should answer "I will XXX", i.e. i will install, I will lint, I will test, I will build etc.

Reply:
ok, done